src/playground/Widgets/TasksList/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { Circle, InProgressTask } from "@/icons"
+import { useMemo } from "react"
+
+type TaskStatus = "in-progress" | "due" | "no-due"
+
+type TaskItemProps = {
+  title: string
+  status: TaskStatus
+}
+
+function TaskItem({ title, status }: TaskItemProps) {
+  return (
+    <div className="flex flex-row items-start gap-2">
+      {(status === "due" || status === "no-due") && (
+        <Circle width={24} opacity={status === "no-due" ? 0.4 : 1} />
+      )}
+      {status === "in-progress" && <InProgressTask />}
+      <p className="mt-[2px] line-clamp-2 flex-1 font-medium">{title}</p>
+    </div>
+  )
+}
+
+interface Props {
+  inProgressTasks: string[]
+  noDueTasks: string[]
+  dueTasks: string[]
+  maxTasksToShow?: number
+}
+
+export function TasksList({
+  inProgressTasks,
+  dueTasks,
+  noDueTasks,
+  maxTasksToShow = 5,
+}: Props) {
+  const tasksToRender = useMemo(
```

Comment:
I don't think the design systems needs to have this logic to handle memory, it should focus in UI and be as "dumb" as possible.